addon/templates/components/common/bs-form/element.hbs

```diff
@@ -30,35 +30,12 @@
     (component this.controlComponent
       value=this.value
       id=this.formElementId
-      name=@name
       type=this.controlType
       label=@label
-      placeholder=placeholder
-      autofocus=autofocus
       disabled=disabled
       readonly=readonly
       required=required
```

Comment:
Can't we remove `readonly` and `required` as well?

Reply:
`disabled` and `readonly` are not that simple cases cause they allow a consumer to set that HTML attributes for all controls of a form at once, e.g.

```hbs
<BsForm @disabled={{true}} @readonly={{true}} as |form|>
  ...
</BsForm>
```

I'm tending to also remove that functionality but would prepare to discuss it separately.

`required` argument wasn't deprecated yet. Will open a PR to deprecate it for `v3` and remove it here.

Reply:
> `required` argument wasn't deprecated yet. Will open a PR to deprecate it for `v3`

See #987